src/fuzzfetch/fetch.py

```diff
@@ -1101,7 +1101,7 @@ def _write_fuzzmanagerconf(self, path):
         output.add_section("Main")
         output.set("Main", "platform", self.moz_info["processor"].replace("_", "-"))
         output.set("Main", "product", f"mozilla-{self._branch}")
-        output.set("Main", "product_version", f"{self.id:>.8}-{self.changeset:>.12}")
+        output.set("Main", "product_version", f"{self.id}-{self.changeset}")
```

Comment:
`{:.8}` and `{:.12}` should remain, just the alignment specifiers (`>`/`<`) weren't needed